circle.yml

```diff
@@ -0,0 +1,14 @@
+machine:
+  node:
+    version: v0.10.40
+
+
+dependencies:
+  pre:
+    - npm install -g npm@3
+
+test:
+  override:
+    - ./node_modules/.bin/eslint --debug . --format tap | ./node_modules/.bin/tap-xunit > ${CIRCLE_TEST_REPORTS}/lint.xml && test ${PIPESTATUS[0]} -eq 0
```

Comment:
Checking pipestatus is needed so build fails on lint error. Checked failed build here: https://circleci.com/gh/cloverfield-tools/universal-react-boilerplate/72